src/palace.pyx

```diff
@@ -2323,7 +2323,8 @@ cdef class ChorusEffect(BaseEffect):
     Parameters
     ----------
     waveform : str
-        
+        Either 'sine' or 'triangle'.
+        By default 'triangle' is used.
```

Comment:
Sorry was forgetting that the default value is in the signature of `__init__`, i.e. we don't need the second sentence.